lib/Common/Core/ConfigParser.cpp

```diff
@@ -337,19 +337,70 @@ void ConfigParser::ParseConfig(HANDLE hmod, CmdLineArgsParser &parser)
         {
             return;
         }
+
         if (_wfopen_s(&configFile, filename, _u("r, ccs=UNICODE")) != 0 || configFile == nullptr)
         {
             return;
         }
     }
+#else
+    // Two-pathed for a couple reasons
+    // 1. PAL doesn't like the ccs option passed in.
```

Comment:
heh, you forced me to debug this- I understand what's going on now. 
- _wfopen_s's implementation [here](https://github.com/Microsoft/ChakraCore/blob/master/pal/src/cruntime/file.cpp#L358) limits the mode to 10 chars (!) - so calling _wfopen with "r, ccs=UNICODE" fails since the mode string is longer than 10 chars
- If I fix that issue, then I hit the issue that I was expecting because of [this issue in MapFileOpenModes](https://github.com/Microsoft/ChakraCore/blob/master/pal/src/cruntime/file.cpp#L123) - D is not supported (lowercase-d would be ignored I think). That's probably why ccs=Unicode might have worked?

I'm leaning on just keeping the two different calls as is and not touching the PAL. Thoughts?